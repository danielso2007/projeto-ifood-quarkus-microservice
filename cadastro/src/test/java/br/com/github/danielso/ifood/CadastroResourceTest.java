package br.com.github.danielso.ifood;

import io.quarkus.test.junit.QuarkusTest;

@QuarkusTest
public class CadastroResourceTest {

}