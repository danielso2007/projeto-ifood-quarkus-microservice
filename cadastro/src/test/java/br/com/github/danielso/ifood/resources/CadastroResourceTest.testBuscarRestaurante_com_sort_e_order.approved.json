[
  {
    "id": 10,
    "dataAtualizacao": "2021-01-01T00:00:00",
    "dataCriacao": "2021-01-01T00:00:00",
    "cnpj": "68454478000125",
    "nome": "A Casa do Porco",
    "proprietario": "Betina Laura JÃ©ssica Almada"
  },
  {
    "id": 20,
    "dataAtualizacao": "2021-01-01T00:00:00",
    "dataCriacao": "2021-01-01T00:00:00",
    "cnpj": "68454478000125",
    "nome": "Olympe",
    "proprietario": "Hugo Ricardo Carlos Pinto"
  }
]